e2e/academic-honesty.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { execSync } from 'child_process';
+import { test, expect } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+test.describe('When the user has not accepted the Academic Honesty Policy', () => {
+  test.use({ storageState: 'playwright/.auth/development-user.json' });
+
+  test.beforeAll(() => {
```

Comment:
```suggestion
  test.beforeEach(() => {
```
See below for reasoning.